ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -262,6 +256,19 @@ - (void)validatedAcquireToken:(ADAuthenticationCallback)completionBlock
     [self requestToken:completionBlock];
 }
 
+- (void)getAccessToken:(ADAuthenticationCallback)completionBlock
+{
+    [[ADTelemetry sharedInstance] startEvent:[self telemetryRequestId] eventName:AD_TELEMETRY_EVENT_ACQUIRE_TOKEN_SILENT];
```

Comment:
Which authority will this use? (developer provided one?)
If we'll use a developer provided authority here, it will break the scenario for sovereign clouds + broker, when the broker returned authority is different from developer provided.

Reply:
If the broker provides the "vt" flag then this code will be bypassed, I'd rather rely on that then modify this code path further.

Reply:
I see, I'll add the "vt" flag then. What does it stand for?

Reply:
valid tenant